integrationtests/src/test/java/org/neo4j/storeupgrade/StoreUpgradeIntegrationTest.java

```diff
@@ -265,10 +265,10 @@ public void migratingOlderDataAndThanStartAClusterUsingTheNewerDataShouldWork()
         @Parameterized.Parameters( name = "{0}" )
         public static Collection<String[]> parameters()
         {
-            return Arrays.<String[]>asList(
+            return Arrays.asList(
```

Comment:
Aah, this test makes sure that we fail if we try to migrate any of those stores.